src/Process/Builder/MutantProcessFactory.php

```diff
@@ -58,7 +58,7 @@ class MutantProcessFactory
     // TODO: is it necessary for the timeout to be an int?
     public function __construct(
         TestFrameworkAdapter $testFrameworkAdapter,
-        int $timeout,
+        float $timeout,
```

Comment:
I would not do that change in this PR and require the `Configuration::getTimeOut()` to return a `float`, but if you wish to I can do it

Reply:
It doesn't require. PHP will be happy to [cast int to float for us](https://3v4l.org/pHN1K).